GameLogic/engine/StandardGame.cs

```diff
@@ -6,180 +6,139 @@
 
 namespace GameLogic
 {
-    public class StandardGame : IStandardGameLogic, IFENSerializable<StandardGameState>
+    public class StandardGame : IStandardGameLogic<StandardGame>
```

Comment:
If `IStandardGameLogic` is renamed to `IGameLogic`, then this line will become

```
public class StandardGame : IGameLogic<StandardGame>
```

Reply:
Renamed!